neo.UnitTests/UT_MemoryPool.cs

```diff
@@ -296,7 +296,7 @@ private void verifyTransactionsSortedDescending(IEnumerable<Transaction> transac
                     if (lastTransaction.FeePerByte == tx.FeePerByte)
                     {
                         if (lastTransaction.NetworkFee == tx.NetworkFee)
-                            lastTransaction.Hash.Should().BeGreaterThan(tx.Hash);
+                            lastTransaction.Hash.Should().BeLessThan(tx.Hash);
```

Comment:
Less or equals perhaps? I didn't see the exact case being tested.

Reply:
tx hashes that collide are filtered at another level. Also, you cannot have two items in a set that exactly match in comparison, so no need for equal also.

Reply:
that's correct.